src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
               }
             }
           }
+        },
+        "valuationComparisonData": {
+          "type": "array",
+          "title": "Valuation Comparison Data",
+          "description": "A collection of data used to compare the valuations (market values) of properties in the real estate or property market. It typically includes various types of information about properties, such as recently sold properties, properties currently on sale, property pricing estimates, property characteristics, and more.",
```

Comment:
I would remove the 'it typically contains' bit - it's what the schema is for!

Reply:
resolved